app/src/main/java/com/example/teamyuml/decentworkmobile/views/CustomListView.java

```diff
@@ -0,0 +1,56 @@
+package com.example.teamyuml.decentworkmobile.views;
+
+import android.app.Activity;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.ArrayAdapter;
+import android.widget.TextView;
+import android.widget.Toast;
+
+import com.example.teamyuml.decentworkmobile.R;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+
+/**
+ * Create own custom ListView style
+ */
+public class CustomListView extends ArrayAdapter<HashMap<String, String>> {
```

Comment:
I think this is a little misguiding. I mean you inherit from ArrayAdapter not an ListView so IMO its rather CustomAdapterListView rather than CustomListView.